? OPCODE/Debug
? OPCODE/Opcode.vcproj
? OPCODE/Release
? VC6/Debug
? VC6/Release
? VC6/msvcdefs.def
? VC6/ode.ncb
? VC6/ode.vcproj
? VC6/Samples/BuildLog.htm
? VC6/Samples/DrawStuff.vcproj
? VC6/Samples/DrawStuff_Debug
? VC6/Samples/DrawStuff_Release
? VC6/Samples/MakeAllTests.ncb
? VC6/Samples/MakeAllTests.sln
? VC6/Samples/MakeAllTests.suo
? VC6/Samples/MakeAllTests.vcproj
? VC6/Samples/Test_BoxStack.vcproj
? VC6/Samples/Test_BoxStack_Debug
? VC6/Samples/Test_BoxStack_Release
? VC6/Samples/Test_Buggy.vcproj
? VC6/Samples/Test_Chain1.vcproj
? VC6/Samples/Test_Chain2.vcproj
? VC6/Samples/Test_Collision.vcproj
? VC6/Samples/Test_Friction.vcproj
? VC6/Samples/Test_Hinge.vcproj
? VC6/Samples/Test_I.vcproj
? VC6/Samples/Test_Joints.vcproj
? VC6/Samples/Test_MovingTrimesh.vcproj
? VC6/Samples/Test_Ode.vcproj
? VC6/Samples/Test_Slider.vcproj
? VC6/Samples/Test_Space.vcproj
? VC6/Samples/Test_Step.vcproj
? VC6/Samples/Test_Trimesh.vcproj
? VC6/Samples/Test_Trimesh_Debug
? VC6/Samples/Test_Trimesh_Release
? VC6/Samples/state.dif
? include/ode/config.h
? ode/test/test_trimesh_convex.cpp
? ode/test/text_convex.cpp
Index: include/ode/collision.h
===================================================================
RCS file: /cvsroot/opende/ode/include/ode/collision.h,v
retrieving revision 1.9
diff -u -r1.9 collision.h
--- include/ode/collision.h	19 Jun 2004 15:44:22 -0000	1.9
+++ include/ode/collision.h	6 Nov 2005 23:06:52 -0000
@@ -84,6 +84,7 @@
   dRayClass,
   dGeomTransformClass,
   dTriMeshClass,
+  dConvexClass,
 
   dFirstSpaceClass,
   dSimpleSpaceClass = dFirstSpaceClass,
Index: ode/src/collision_kernel.cpp
===================================================================
RCS file: /cvsroot/opende/ode/ode/src/collision_kernel.cpp,v
retrieving revision 1.18
diff -u -r1.18 collision_kernel.cpp
--- ode/src/collision_kernel.cpp	19 Apr 2004 18:27:56 -0000	1.18
+++ ode/src/collision_kernel.cpp	6 Nov 2005 23:10:02 -0000
@@ -146,6 +146,17 @@
   setCollider (dTriMeshClass,dTriMeshClass,&dCollideTTL);
   setCollider (dTriMeshClass,dCCylinderClass,&dCollideCCTL);
 #endif
+
+#ifdef BULLET_CONVEX_SUPPORT
+  ///see collision_convex.cpp
+  setCollider (dConvexClass,dConvexClass,&dCollideConvexConvex);
+  setCollider (dConvexClass,dPlaneClass,&dCollideConvexConvex);
+  setCollider (dConvexClass,dBoxClass,&dCollideConvexConvex);
+  setCollider (dConvexClass,dSphereClass,&dCollideConvexConvex);
+  setCollider (dConvexClass,dCCylinderClass,&dCollideConvexConvex);
+  setCollider( dTriMeshClass,dConvexClass, dCollideConvexConvex);
+#endif //BULLET_CONVEX_SUPPORT
+
   setAllColliders (dGeomTransformClass,&dCollideTransform);
 }
 
Index: ode/src/collision_std.h
===================================================================
RCS file: /cvsroot/opende/ode/ode/src/collision_std.h,v
retrieving revision 1.2
diff -u -r1.2 collision_std.h
--- ode/src/collision_std.h	1 Dec 2002 06:13:42 -0000	1.2
+++ ode/src/collision_std.h	6 Nov 2005 23:03:52 -0000
@@ -37,6 +37,7 @@
 // the same interface as dCollide(). the first and second geom arguments must
 // have the specified types.
 
+
 int dCollideSphereSphere (dxGeom *o1, dxGeom *o2, int flags,
 			  dContactGeom *contact, int skip);
 int dCollideSphereBox (dxGeom *o1, dxGeom *o2, int flags,
@@ -63,6 +64,9 @@
 			  int flags, dContactGeom *contact, int skip);
 int dCollideRayPlane (dxGeom *o1, dxGeom *o2, int flags,
 		      dContactGeom *contact, int skip);
+///dCollideConvexConvex: see collision_convex.cpp for details
+int dCollideConvexConvex(dxGeom *o1, dxGeom *o2, int flags,
+			  dContactGeom *contact, int skip);
 
 
 #endif
Index: ode/src/collision_trimesh_internal.h
===================================================================
RCS file: /cvsroot/opende/ode/ode/src/collision_trimesh_internal.h,v
retrieving revision 1.10
diff -u -r1.10 collision_trimesh_internal.h
--- ode/src/collision_trimesh_internal.h	21 Sep 2004 20:54:21 -0000	1.10
+++ ode/src/collision_trimesh_internal.h	6 Nov 2005 23:04:52 -0000
@@ -25,6 +25,7 @@
 #ifndef _ODE_COLLISION_TRIMESH_INTERNAL_H_
 #define _ODE_COLLISION_TRIMESH_INTERNAL_H_
 
+int dCollideTrimeshConvex(dxGeom* g1, dxGeom* cnvx, int Flags, dContactGeom* Contacts, int Stride);
 int dCollideSTL(dxGeom *o1, dxGeom *o2, int flags, dContactGeom *contact, int skip);
 int dCollideBTL(dxGeom *o1, dxGeom *o2, int flags, dContactGeom *contact, int skip);
 int dCollideRTL(dxGeom *o1, dxGeom *o2, int flags, dContactGeom *contact, int skip);
